ci/test-upgrade/fetch-cli-artifacts.sh

```diff
@@ -32,11 +44,18 @@ fetch_cli_artifact() {
 
     mkdir -p ${dir}
 
-    if [ -f "${artifact}" ]; then
-        cp ${artifact} ${dir}
-    else
-        fetch_cli_s3_artifact ${artifact} ${dir}
-    fi
+    case "${artifact}" in
```

Comment:
`case "${artifact}" in` oh is that a magic string prefix check? :D

Reply:
Yep. Bash has some nice features [built in](https://tldp.org/LDP/Bash-Beginners-Guide/html/sect_07_03.html).

Reply:
Oh I see, this is generic pattern matching!